Sources/SwiftCurrent_SwiftUI/UIKitInterop/HostedWorkflowItem.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  HostedWorkflowItem.swift
+//  SwiftCurrent_SwiftUI
+//
+//  Created by Tyler Thompson on 8/7/21.
+//  Copyright © 2021 WWT and Tyler Thompson. All rights reserved.
+
+#if (os(iOS) || os(tvOS) || targetEnvironment(macCatalyst)) && canImport(UIKit)
+import UIKit
+import SwiftUI
+import SwiftCurrent
+
+/**
+ A wrapper around `UIHostingController` that is `FlowRepresentable`.
+
+ ### Discussion
+ `HostedWorkflowItem` is designed to be used in UIKit workflows that want to interoperate with SwiftUI. You do not need to inherit from this class, nor do you need to reference it outside of your workflow creation.
+
+ #### Example
+ ```swift
+ struct SwiftUIView: View, FlowRepresentable {
+     weak var _workflowPointer: AnyFlowRepresentable?
+
+     var body: some View {
+         Text("My View")
+     }
+ }
+
+ // from the UIViewController launching the Workflow:
+ launchInto(Workflow(FlowRepresentableViewController.self)
+         .thenProceed(with: HostedWorkflowItem<SwiftUIView>.self))
+ ```
+ */
+@available(iOS 14.0, macOS 11, tvOS 14.0, *)
+public final class HostedWorkflowItem<Content: FlowRepresentable & View>: UIHostingController<Content>, FlowRepresentable {
```

Comment:
Should this passthrough the `viewDidLoad` of the view FlowRepresentable as well?

Reply:
`View`s in SwiftUI do not have a `viewDidLoad` method, and `FlowRepresentable` has no method I could call when `viewDidLoad` happens. Can you expand on what you mean?

Reply:
Well I apparently can't word today.  I meant `shouldLoad`.  The `shouldLoad` should passthrough.

Reply:
WOW those words helped a bunch, thank you! It is done.